File: app/src/main/java/com/task/paginationlist/presentation/piclist/views/adapters/holders/LoadingHolder.java
package com.task.paginationlist.presentation.piclist.views.adapters.holders;

import android.support.v7.widget.RecyclerView;
import android.view.View;

import com.task.paginationlist.R;
import com.task.paginationlist.presentation.piclist.views.paginator.LoaderView;

import butterknife.BindView;
import butterknife.ButterKnife;

public class LoadingHolder extends RecyclerView.ViewHolder {

    @BindView(R.id.footer_list_item)
    public LoaderView loaderView;

    public LoadingHolder(View itemView) {
        super(itemView);
        ButterKnife.bind(this, itemView);
    }
}

File: app/src/main/java/com/task/paginationlist/presentation/piclist/views/adapters/holders/LoadingHolder.kt
package com.task.paginationlist.presentation.piclist.views.adapters.holders

import android.support.v7.widget.RecyclerView
import android.view.View
import com.task.paginationlist.presentation.piclist.views.paginator.LoaderView
import kotlinx.android.synthetic.main.footer_item.view.*

class LoadingHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {

    var loaderView: LoaderView? = itemView.footerListItem

}
